src/components/common.js

```diff
@@ -351,10 +351,17 @@ export const AsyncCreatableSelect = props => {
   return h(RAsyncCreatableSelect, _.merge(commonSelectProps, props))
 }
 
-export const PageBox = ({ children, style = {}, ...props }) => {
+export const PageBoxVariants = {
+  LIGHT: 'light'
+}
+
+export const PageBox = ({ children, variant = undefined, style = {}, ...props }) => {
```

Comment:
`undefined` is the default value for anything, so defining it as the default here is a no-op.
```suggestion
export const PageBox = ({ children, variant, style = {}, ...props }) => {
```